pages/profile/views/index.jsx

```diff
@@ -13,7 +13,7 @@ import Modules from '@asl/pages/pages/task/read/views/modules';
 class Index extends React.Component {
 
   renderAsruAssociation (establishments) {
-    return (<ul className="panel-list">
+    return (<ul className="panel-list">Date of birth
```

Comment:
Why have the words "Date of birth" appeared here? That seems wrong.

Reply:
Must have copied it by mistake, removed.